prvni-krucky/promenne/extra-excs/antikvariat.md

```diff
@@ -0,0 +1,11 @@
+---
+title: Antikvariát
+demand: 2
+---
+
+Antikvariát *Poslední strana* nabízí veškeré své tituly za 50 kč.
+
+1. Uložte název antikvariátu do libovolné proměnné.
+2. Cenu za jeden titul uložte do proměnné `cena_za_kus`.
+3. Antikvariát má dnes speciální akci, -30% na všechny dostupné tituly! Slevu uložte do proměnné `sleva` jako desetinné číslo.
+4. Propočtěte cenu za kus na základě této slevy.
```

Comment:
Běžnější výraz v češtině je _Vypočtěte_.

Taky řazený seznam v Markdownu je lepší psát pouze s `1.`:
```
1.
1.
1.
```
a vykreslí se to jako

1.
1.
1.

resp. kodim.cz to vykreslí jako
a.
b.
c.

Zdroják:
https://github.com/Czechitas-Python/uvod-do-progr-1/blob/main/prvni-krucky/funkce-moduly/excs/klasicke-zaokrouhlovani.md?plain=1

Vykreslení:
https://kodim.cz/kurzy/uvod-do-progr-1/prvni-krucky/funkce-moduly/cteni-na-doma#excs%3Eklasicke-zaokrouhlovani

Zdroják:
https://github.com/Czechitas-Python/uvod-do-progr-1/blob/main/prvni-krucky/cykly/excs/hry.md?plain=1

Vykreslení:
https://kodim.cz/kurzy/uvod-do-progr-1/prvni-krucky/cykly/excs#excs%3Ehry

Příklady jsou super. Díky moc!

Reply:
Ďakujem za opravu, to vykreslovanie na kodim.cz ma práve pomýlilo, tak do budúcnosti už budem vedieť. :)